Aptoide/aptoide-client-v8
# [feature/MOB-555] Secure login blog link

**What does this PR do?**

   Redirects to blog post when the user clicks on Secure login text.

**Database changed?**

   No

**How should this be manually tested?**

  Go to send magic link view and click on "Secure Login - ...". It should redirect you to a blog post about it.

**What are the relevant tickets?**

  [MOB-555](https://aptoide.atlassian.net/browse/MOB-555)

**Code Review Checklist**
